pipeline/context_explorer/get_context_analysis.py

```diff
@@ -132,41 +135,51 @@ def load_all_data(
     repurposing_table_path,
     oncref_table_path,
 ):
+
+    all_data_dict = dict()
+    datasets_to_test = dict()
+    data_for_extra_cols = dict()
+
     tc = create_taiga_client_v3()
     subtype_tree, context_matrix = load_subtype_tree(
         tc=tc,
         subtype_tree_taiga_id=subtype_tree_taiga_id,
         context_matrix_taiga_id=context_matrix_taiga_id,
     )
+    all_data_dict["subtype_tree"] = subtype_tree
+    all_data_dict["context_matrix"] = context_matrix
+
     gene_effect, gene_dependency = load_crispr_data(
         tc=tc,
         gene_effect_taiga_id=gene_effect_taiga_id,
         gene_dependency_taiga_id=gene_dependency_taiga_id,
     )
+    datasets_to_test["CRISPR"] = gene_effect
+    data_for_extra_cols["gene_dependency"] = gene_dependency
+
     rep_sensitivity = load_prism_data(
         tc=tc,
         repurposing_matrix_taiga_id=repurposing_matrix_taiga_id,
         repurposing_list_taiga_id=repurposing_list_taiga_id,
     )
+    datasets_to_test["PRISMRepurposing"] = rep_sensitivity
 
-    oncref_aucs, oncref_log_aucs = load_oncref_data(
-        tc=tc, oncref_auc_taiga_id=oncref_auc_taiga_id
-    )
+    if oncref_auc_taiga_id is not None:
+        oncref_aucs, oncref_log_aucs = load_oncref_data(
+            tc=tc, oncref_auc_taiga_id=oncref_auc_taiga_id
+        )
+        datasets_to_test["PRISMOncRef"] = oncref_log_aucs
+        data_for_extra_cols["oncref_aucs"] = oncref_aucs
 
     selectivity_vals = format_selectivity_vals(
-        pd.read_csv(repurposing_table_path), pd.read_csv(oncref_table_path)
+        repurposing_table_path, oncref_table_path
     )
+    data_for_extra_cols["selectivity_vals"] = selectivity_vals
 
-    return {
-        "subtype_tree": subtype_tree,
-        "context_matrix": context_matrix,
-        "gene_effect": gene_effect,
-        "gene_dependency": gene_dependency,
-        "rep_sensitivity": rep_sensitivity,
-        "oncref_log_aucs": oncref_log_aucs,
-        "oncref_aucs": oncref_aucs,
-        "selectivity_vals": selectivity_vals,
-    }
+    all_data_dict["datasets_to_test"] = datasets_to_test
+    all_data_dict["data_for_extra_cols"] = data_for_extra_cols
```

Comment:
I'm a little confused about what "data_for_extra_cols" is supposed to be doing. Is this something that was needed to handle Oncref being None for public?